.github/workflows/ci.yml

```diff
@@ -18,13 +27,7 @@ jobs:
           java-version: '8'
           distribution: 'temurin'
 
-      - name: Build a Fat JAR
```

Comment:
We still want the fat jar step. Otherwise we aren't uploading an artifact that includes all the necessary dependencies. We can run the tests before building the fat jar as I don't the tests depend on the fat jar.

Reply:
I agree, because tests passing is not enough to guarantee that the build will be successful. Hence, I added the build step.